docs/config.yaml

```diff
@@ -113,19 +115,27 @@ s3:
 # OPTIONAL: container registry-related keys
 registry_repos:
   # OPTIONAL: repo to which to push oscontainer
-  oscontainer: quay.io/fedora/fedora-coreos
+  oscontainer:
+	registry: quay.io/fedora/fedora-coreos
+    tags: ["${STREAM}"]
```

Comment:
Should add `REQUIRED` markers on these and docstrings for consistency.

Reply:
added REQUIRED docstrings